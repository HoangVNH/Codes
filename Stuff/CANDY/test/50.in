10
83
XDDXXXXDVDDDVVDVXDVXXXVXDVVDXVXVVVXXDDDVDXVXVDXXDVXXDXDXDXXXVXXDXVXDXXXXVDXXVDXDXVV
96
XXDDXXXVDXVXDDDVVDVVVDDXVXXDDXVDXVDVVDDVDXVDDDDXVVXVDVVXVDXDDXDXDVDXXXXVXVXVDVXDVVDXVDVXDXXVXDXD
75
DDVDDXXDXXDVDXXDVXDVVVVVXVDVXVDDVVVDDDXDDVVVXXVDVXVXVVVVVDVXDXVVVXVVVVVDVXD
100
XDXXXDXVVDVVDDXXVDXDXVVVDVXVDVDXXVDXDDXVVDXXVXDVDXDXXVXXDDVVXVXVXDXXVVDXVVDXXXXDXVVVXDVXXVVXDXXXVVXV
83
DVVDXXDVXXVDVVDVXDVDXDVDXDXVVVVDDVVXVVVVDXVDVXXVVVDDXDVVXDXXVXDXDVXDVVDVVXVDXVXDXDV
87
XXDVXDXDDXVXDDVVVVXVDXVVVVXVXXDDVXDDDXXDVXDXDDDVVXXVXXDVVXVDDXXDDDDVDDXXVXXXVXVDXVVDDXX
85
DDDXVXXXVDDXXVVVVXXXDVVDVXDDVVDXDVXVDXVXVXXXDXDXDDDVXVVXVDVXXVDVXVVXDDVXVDVXVXVDVVDVD
72
VDXDXVDXXDXDVXDXDXXDVXDDVDDXVXDVVXXVDDDDXDDDVVXVVXXVXDDXVVVVXDXDXDVXVVXX
73
VXXXDDVXVVVXXDDVXVXXDDVDVXVXXVDDXDDVVDVXVVVXXDVVDDXDXDDVDXVDVVDDDXVXXXXXD
75
VDXDXXVVXXXDDXVDVDDXVDVXVVXXVDXDXDXXVVVVVXVXVXDXXVDDDDVXVXDVDXDXXDVVXVVXDDX
