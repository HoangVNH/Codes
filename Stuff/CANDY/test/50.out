82
94
74
99
81
86
84
71
72
74
